src/errors.rs

```diff
@@ -0,0 +1,27 @@
+use std::fmt;
+use std::error::Error;
+
+#[derive(Debug)]
+pub struct FontCreationError {
```

Comment:
You might want to use the [failure](https://boats.gitlab.io/failure/) crate and derive `Fail` :)